scripts/works.js

```diff
@@ -135,21 +144,180 @@ btn0.addEventListener('click', () => {
+  li33.className = 'title-and-year';
+  li33.textContent = `${data[2].technologies[0].teck3}`;
+  div2.className = 'live-and-source-btn';
+  seeLive.className = 'btns';
+  seeSource.className = 'btns';
+  seeLive.textContent = 'See Live';
+  seeSource.textContent = 'See Source';
+  liveIcon.setAttribute('src', '../assets/live-icon.png');
+  liveIcon.className = 'live-icon';
+  seeLive.append(liveIcon);
+  githubIcon.setAttribute('src', '../assets/github-icon.png');
+  githubIcon.className = 'github-icon';
+  seeSource.append(githubIcon);
+  ul2.append(li11, li22, li33);
+  ul1.append(li1, li2, li3);
+  div2.append(seeLive, seeSource);
+  div1.append(ul2, div2);
+  div.append(p, div1);
+  popUp2.append(h2, ul1, image, image2, div, popUpTogglerBtn);
+  body.appendChild(popUp2);
 });
 btn3.addEventListener('click', () => {
-  popUp3.classList.toggle('show');
+  body.classList.add('show');
+  popUpTogglerBtn.className = 'toggle-to';
+  popUpTogglerBtn.textContent = 'X';
+  popUpTogglerBtn.setAttribute('href', `#pop-up${3}`);
+  popUp3.classList.add('show');
+  popUpBg.classList.add('show');
+  window.scrollTo(0, 0);
+  image.className = 'mobile';
+  image.setAttribute('src', `${data[3].image[0].imageSrc}`);
+  image.setAttribute('alt', `${data[3].image[0].imageAlt}`);
+  image.className = 'mobile';
+  image2.setAttribute('src', `${data[3].image[1].imageSrc}`);
+  image2.setAttribute('alt', `${data[3].image[1].imageAlt}`);
+  image2.className = 'desktop';
+  div1.className = 'techs-and-buttons';
+  div.className = 'project-description-container-popup';
+  h2.className = 'project-title';
+  h2.textContent = `${data[3].name}`;
+  ul1.className = 'canopy-list';
+  li1.className = 'canopy';
+  li1.textContent = `${data[3].featured[0].feature1}`;
+  li2.className = 'title-and-year';
+  li2.textContent = `${data[3].featured[0].feature2}`;
+  li3.className = 'title-and-year';
+  li3.textContent = `${data[3].featured[0].year}`;
+  p.className = 'project-description';
+  p.textContent = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent';
+  ul2.className = 'teck-stacks';
+  li11.className = 'canopy';
+  li11.textContent = `${data[3].technologies[0].teck1}`;
+  li22.className = 'title-and-year';
+  li22.textContent = `${data[3].technologies[0].teck2}`;
+  li33.className = 'title-and-year';
+  li33.textContent = `${data[3].technologies[0].teck3}`;
+  div2.className = 'live-and-source-btn';
+  seeLive.className = 'btns';
+  seeSource.className = 'btns';
+  seeLive.textContent = 'See Live';
+  seeSource.textContent = 'See Source';
+  liveIcon.setAttribute('src', '../assets/live-icon.png');
+  liveIcon.className = 'live-icon';
+  seeLive.append(liveIcon);
```

Comment:
- [ ] :thinking: I suggest using avoiding to repeat yourself implementing the onlick event at each 'see-project' in order to stick to meet the javascript best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=simplicity.-,DRY.,-If%20you%20use) 

HINT: for that you can create a loop for these buttons and take the from the project array data by the button index